packages/notifier/src/notifier.js

```diff
@@ -318,7 +188,7 @@ export const makeNotifierFromAsyncIterable = asyncIterableP => {
   const notifier = Far('notifier', {
     // Don't leak the original asyncIterableP since it may be remote and we also
     // want the same semantics for this exposed iterable and the baseNotifier.
-    ...makeAsyncIterableFromNotifier(baseNotifier),
+    ...subscribeLatest(baseNotifier),
     ...baseNotifier,
 
     /**
```

Comment:
`getSharableNotifierInternals` is already provided by `makeNotifier`.
```suggestion
    ...makeNotifier(baseNotifier),
    ...baseNotifier,
```